backend/views/core/clients/create.py

```diff
@@ -25,35 +25,38 @@ def create_client(request: HtmxHttpRequest):
         messages.error(request, error)
         return redirect("clients create")
 
-    if request.user.logged_in_as_team:
-        client = Client.objects.create(
-            organization=request.user.logged_in_as_team,
-        )
+    client = create_client_instance(request.user, client_details)
+
+    if not client:
+        messages.error(request, "Failed to create client - an unknown error occurred")
     else:
-        client = Client.objects.create(
-            user=request.user,
-        )
+        messages.success(request, f"Client created successfully (#{client.id})")
 
-    for model_field, new_value in client_details.items():
-        setattr(client, model_field, new_value)
+    return redirect("clients dashboard")
 
-    client.save()
 
-    if client:
-        messages.success(request, f"Client created successfully (#{client.id})")
-    else:
-        messages.error(request, "Failed to create client - an unknown error occurred")
-    return redirect("clients dashboard")
+def create_client_instance(user, client_details):
+    organization = user.logged_in_as_team if user.logged_in_as_team else None
```

Comment:
This needs to support adding customers under an individual user aswell as organisation